dask_expr/_expr.py

```diff
@@ -1303,6 +1303,17 @@ def operation(df):
         return df.copy(deep=True)
 
 
+class ToBackend(Elemwise):
+    _parameters = ["frame", "options"]
+    _projection_passthrough = True
+    _filter_passthrough = True
+    _preserves_partitioning_information = True
+
+    @staticmethod
+    def operation(df, options):
+        raise NotImplementedError()
```

Comment:
Can you set operation = None, this should raise during graph construction, not execution IIRC

Reply:
Sounds good

Reply:
Okay - Looks like `Elemwise` already has `operation = None` by default. So, I just removed the operation def altogether. Thanks!